Cutout/README.md

```
@@ -51,13 +51,16 @@ public static partial class MyTemplate
 
 ## Template Language
 
-Everything that is not between `{{` and `}}` is treated as a string literal.
-any expression that does not start with a known keyword is treated as a variable
-to be rendered.
+Everything that is not between `{{` or `{@` and `}}` or `%}` is treated as a 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)